### ISSUE TITLE
Add missing captions to figures

### ISSUE THREAD

**Issue by brunerae** [uswds/uswds-site#1371]
@brunerae / @mikki-stacey - add a list of figures here that need captions

**Comment by thisisdano** [comment]
Is this done?

**Comment by brunerae** [comment]
@thisisdano during the page audit, Mikki [noted here](https://docs.google.com/spreadsheets/d/1xhshfcXHI3u0UCQFTl5alWbELJovedf9/edit#gid=1262792509) which pages having figures missing captions. See "page audit" tab.

**Comment by brunerae** [comment]
This issue can be closed based on above comment, so assigning to Dan.

**Comment by thisisdano** [comment]
Let's use this issue to add the missing captions. Let's add captions to the images on the following pages:
https://designsystem.digital.gov/maturity-model/
https://designsystem.digital.gov/documentation/fundamentals/

**Comment by bonnieAcameron** [comment]
I've updated the code block from <div> to <figure> (after a discussion with Amy about using captions in markdown. I've also added a caption to further explain or interpret the image (which already includes alt text)
